### ISSUE TITLE
Is there still support for this programm?

### ISSUE THREAD

**Issue by n0valis** [sabaatworld/HyperionScreenCap#52]
Hi,
I have noticed that there was less activity from the programmer lately. Neither in the code section itself nor in the issues section.
Will this programm going to be further developed in the future?
It seems to me that it has several issues that needed to be addressed to make it work properly.

**Comment by sabaatworld** [comment]
I try to work on it from time to time. Last major thing I added was FlatBuffer support.

I'd be happy to accept pull requests if you were to fix something.

**Comment by n0valis** [comment]
Appreciate the offer, but I do not have those skills.
I love the programm,but unfortunately it is not stable in conjunction with Hyperion. Especially the flatbuffer.
Hopefully you will not stop to maintain. Thanks.

**Comment by sabaatworld** [comment]
I was under the impression that flatbuffer worked as well as protobuffer. Have you noticed any specific problems with it?

**Comment by n0valis** [comment]
yes I have. It works until I start Kodi. As I am using Kodi as my main media player, as soon as I start a video, the connection is lost. I have tp leave Kodi , can stop and restart grabbing but then evtentually as soon as Kodi is fullscreen the connection is lost again.
It's the switching. HDR also seems to make problems.

**Comment by sabaatworld** [comment]
Does it switch to different resolution in full screen that the connection is lost?

Also does it work with proto?

**Comment by Dinth** [comment]
same thing happens to me, especially with HDR vids (logged a separate ticket about that). But had same issue with protobuffer, even if it was not happening as often, it was happening often enough.

**Comment by sabaatworld** [comment]
Newly released v2.9 has modified retry behavior which might fix this issue.
